Find a functionally equivalent code:android.app.backup.BackupManager.beginRestoreSession:COMMENT
Method Modifier: public      hidden      
Comment:/**
 * Begin the process of restoring data from backup.  See the
 * {@link android.app.backup.RestoreSession} class for documentation on that process.
 * @hide
 */

@@ -9,15 +9,15 @@
                 session = new RestoreSession(mContext, binder);
             }
         } catch (RemoteException e) {
-            Log.w(TAG, "beginRestoreSession() couldn't connect");
+            Log.e(TAG, "beginRestoreSession() couldn't connect");
         }
     }
     return session;

